distribution.pm

```diff
@@ -129,7 +129,7 @@ sub script_run {
     my %args = testapi::compat_args(
         {
             timeout => $bmwqemu::default_timeout,
-            quiet   => undef
+            quiet   => testapi::get_var('QUIET', 0)
```

Comment:
add the variable to backend_vars.md (https://github.com/os-autoinst/os-autoinst/blob/master/doc/backend_vars.asciidoc) but also in the meantime, rename it to _QUIET_SCRIPT_CALLS

Reply:
done